### PR TITLE
Finish html structure

### PR DESCRIPTION
**Pull request by ey-charles**
# Project 1: Floating and Positioning
Recreating the New York Times website

These are our highlighted milestones:

- Used semantic tags (aside, article, section, footer, header, nav, and main). H1 tag used on line 48.
- Used Flexbox on the following sections: top bar, main article (and Editor's Picks section), 'The Theory of Inflation' article, and site index.
- Used float in the Editor's Picks section
- Used position (fixed) for the topbar
- Used grid for the Site Index

### CODE REVIEW

**Comment by crgc** [comment]
- Used **semantic tags** (aside, article, section, footer, header, nav, and main)
- Used **Flexbox** on the following sections: top bar, main article (and Editor's Picks section), 'The Theory of Inflation' article, and site index.
- Used **float** in the Editor's Picks section
- Used **position** (fixed) for the topbar
- Used **grid** for the Site Index

File: resources/css/main.css [hunk 1]
```diff
@@ -1,37 +1,499 @@
...
+  padding-left: 90px;
+}
+
+.section-content {
+  width: 90%;
+  padding-left: 300px;
+}
+
+/* Main content */
+#article-content {
```
Comment (arslanbisharat) [inline resources/css/main.css]: - [x] Don't use IDs in selectors.

File: resources/css/main.css [hunk 2]
```diff
@@ -1,37 +1,499 @@
...
+}
+
+.inflation-footer-authors {
+  color: rgb(51, 51, 51);
+  font-family: arial, helvetica, sans-serif;
+  font-size: 13px;
+}
+
+/** Site index **/
+#site-index {
```
Comment (arslanbisharat) [inline resources/css/main.css]: - [x] Don't use IDs in selectors.

File: resources/css/main.css [hunk 3]
```diff
@@ -1,37 +1,499 @@
...
+
+.site-index-section-list-item .icon {
+  width: 12px;
+}
+
+.border-bottom {
+  border-bottom: 1px solid #e2e2e2;
+}
+
+#site-footer {
```
Comment (arslanbisharat) [inline resources/css/main.css]: - [x] Don't use IDs in selectors.

File: resources/css/main.css [hunk 4]
```diff
@@ -1,37 +1,499 @@
...
+
+/** Site index **/
+#site-index {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 1200px;
+  margin: 90px auto 10px;
+  border-top: 1px solid #e2e2e2;
+}
+
+#site-index::before {
```
Comment (arslanbisharat) [inline resources/css/main.css]: - [x] Don't use IDs in selectors.

File: resources/css/main.css [hunk 5]
```diff
@@ -1,37 +1,499 @@
...
+  content: "";
```
Comment (arslanbisharat) [inline resources/css/main.css:388]: - [x] You have empty properties

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [ ] **Use semantic HTML**
your HTML pages need H1 tags for semantic, accessibility, and SEO reasons. Read more here and make sure to add the H1 headers.
Check more about H1 tags here https://www.searchenginejournal.com/h1-tag-seo-how-important/387306/#close

- [x] You have a comment in your Pull Request. Please add that comment in your Pull Request Description. 
- [x] I have given some suggestions below. Please work on them 

**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: resources/css/main.css (hunk 5 above)
Comment (crgc) [reply resources/css/main.css:388]: This is intentional. The `content: "";` is meant to create a shallow top border for the site index. It is that way in the New York Times article as well.

File: resources/css/main.css (hunk 1 above)
Comment (ey-charles) [reply resources/css/main.css]: We fixed this by using a class instead. Thank you!

File: resources/css/main.css (hunk 2 above)
Comment (ey-charles) [reply resources/css/main.css]: Done

File: resources/css/main.css (hunk 3 above)
Comment (ey-charles) [reply resources/css/main.css]: We fixed this by using a class instead. Thank you!

File: resources/css/main.css (hunk 4 above)
Comment (ey-charles) [reply resources/css/main.css]: Done

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
#### Congratulations, this project is now approved. You have done well in making the changes that were required. You can now proceed with the next project and good luck.
slack(@ebeagusamuel)

**Pull request merged**